src/cli/abrtcli/filtering.py

```diff
@@ -51,8 +51,15 @@ def filter_paths(problems, paths):
 
     def predicate(problem):
         for path in paths:
```

Comment:
Could you please add a comment or two explaining what the code does? It's a bit confusing without any context at the moment.

Reply:
It just checks whether any of the `problems` matches any of the `paths` where the latter can be contain both globs and paths. The fact that it's a function just makes it possible to use it with `filter()` a few lines later. I was thinking of renaming it to something more descriptive than "predicate" but then I noticed there's several more such functions throughout the file. I suppose I should come up with something universal. Or just rewrite the code without these functions. What do you think?